content/spin/v2/python-components.md

```diff
@@ -205,17 +193,21 @@ Hello from the Python SDK
 This next example will create an outbound request, to obtain a random fact about animals, which will be returned to the calling code. If you would like to try this out, you can go ahead and update your existing `app.py` file from the previous step; using the following source code:
```

Comment:
The description here no longer seems to match the sample.  (My preference would be to keep the old sample which is more focused on the send and response - the new one has extraneous concerns and will have woes with `allowed_outbound_hosts`.)